musicbrainz/getArtists.js

```diff
@@ -138,7 +138,7 @@ module.exports = {
             }
 
 
-        }, 3000)
+        }, 1200)
```

Comment:
Agree with @pfent :) It's defined in a couple locations so having it on top in `const TTLXX` fashion would't be _that_ bad :)